next.config.js

```diff
@@ -32,6 +44,7 @@ module.exports = withBundleAnalyzer({
   // TODO: reactStrictMode with react18 breaks edtr.io atm (inside react-hotkeys)
   reactStrictMode: false,
   productionBrowserSourceMaps: true,
+  transpilePackages: ['ramda'],
```

Comment:
why is this necessary?

Reply:
for context: https://github.com/vercel/next.js/issues/40183

Reply:
thanks!